src/components/views/rooms/RoomList.js

```diff
@@ -45,6 +46,7 @@ module.exports = React.createClass({
     componentWillMount: function() {
         var cli = MatrixClientPeg.get();
         cli.on("Room", this.onRoom);
+        cli.on("deleteRoom", this.onDeleteRoom);
```

Comment:
`Room.whatever` is by convention used to tell people that the property `whatever` on the provided `Room` object has changed. This isn't the case here.